src/views/components/global/AgriTable/index.vue

```diff
@@ -0,0 +1,178 @@
+<template>
+  <div>
+    <md-table v-model="users" md-sort="name" md-sort-order="asc" md-card>
+      <md-table-toolbar>
+        <h1 class="md-title">Users</h1>
+      </md-table-toolbar>
+
+      <md-table-row slot="md-table-row" slot-scope="{ item }">
+        <md-table-cell md-label="ID" md-sort-by="id" md-numeric>{{ item.id }}</md-table-cell>
+        <md-table-cell md-label="Name" md-sort-by="name">{{ item.   name }}</md-table-cell>
+        <md-table-cell md-label="Email" md-sort-by="email">{{ item.email }}</md-table-cell>
+        <md-table-cell md-label="Gender" md-sort-by="gender">{{ item.gender }}</md-table-cell>
+        <md-table-cell md-label="Job Title" md-sort-by="title">{{ item.title }}</md-table-cell>
+      </md-table-row>
+    </md-table>
+  </div>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+
+const name = 'agritable';
+export default Vue.component(name, {
+  name,
+
+  data: () => ({
+    users: [
```

Comment:
The data should probably be passed into this as a prop. We don't want to tie any of these generic global components to a specific data set

Eventually the data will be coming from the store, but it's fine to have some mock data in the vue property `data` in the meantime. Just raise the data up to a different component though